be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -67,8 +67,13 @@ BetaRowsetWriter::~BetaRowsetWriter() {
 Status BetaRowsetWriter::init(const RowsetWriterContext& rowset_writer_context) {
     _context = rowset_writer_context;
     _rowset_meta.reset(new RowsetMeta);
-    if (_context.data_dir) {
+    if (_context.fs == nullptr && _context.data_dir) {
```

Comment:
if context.fs == nullptr and context.data_dir is also nullptr, then what happen?

Reply:
then _rowset_meta->set_fs(nullptr), this is the same with _rowset_meta->set_fs(_context.fs);